blog/2024-04-18-opentofu-1-7-0-beta1.md

```diff
@@ -0,0 +1,225 @@
+      fallback{
+        method = method.unencrypted.migration
+      }
+      ## Enable this after migration:
+      #enforced = true
+    }
+  }
+}
+```
+
+You can migrate back using the following syntax:
+
+```hcl
+terraform {
+  encryption {
+    key_provider "pbkdf2" "my_passphrase" {
+      ## Enter a passphrase here:
+      passphrase = ""
+    }
+
+    method "aes_gcm" "my_method" {
+      keys = key_provider.pbkdf2.my_passphrase
+    }
+
+    method "unencrypted" "migration" {
+    }
+
+    state {
+      method  = method.unencrypted.migration
+      enforced = false
+      fallback{
+        method = method.aes_gcm.my_method
+      }
+    }
+  }
+}
+```
+
+If you have access to an AWS, GCP account, or an OpenBao/MPL-licensed HashiCorp Vault installation, you can also [test these key providers](/docs/language/state/encryption/#key-providers).
+
+## Removed block
+
+The removed block lets you remove a resource from the state file but keep it on the infrastructure. We have prepared a [full documentation](/docs/language/resources/syntax/#removing-resources) for this feature. You can test it by creating a resource first:
+
+```hcl
+resource "local_file" "test" {
+  content = "Hello world!"
+  filename = "test.txt"
+}
+```
+
+After applying, you can replace the resource with a removed block:
+
+```hcl
+removed {
+  from = local_file.test
+}
+```
+
+After the next apply, you will see that the `local_file.test` resource no longer exists in your state file, but the `test.txt` file should still exist on your disk. You can now remove the removed block safely.
+
+## Built-in function changes
+
+This release also contains several new functions and changes to existing functions:
+
+- New function: [templatestring](https://1-7-0-alpha1.opentofu.pages.dev/docs/language/functions/templatestring/)
+- New function: [base64gunzip](https://1-7-0-alpha1.opentofu.pages.dev/docs/language/functions/base64gunzip/)
+- New function: [cidrcontains](https://1-7-0-alpha1.opentofu.pages.dev/docs/language/functions/cidrcontains/)
+- New function: [urldecode](https://1-7-0-alpha1.opentofu.pages.dev/docs/language/functions/urldecode/)
+- New function: [issensitive](https://1-7-0-alpha1.opentofu.pages.dev/docs/language/functions/issensitive/)
+- [nonsensitive](https://1-7-0-alpha1.opentofu.pages.dev/docs/language/functions/nonsensitive/) no longer returns an error when the applied values are not sensitive.
+- [templatefile](https://1-7-0-alpha1.opentofu.pages.dev/docs/language/functions/templatefile/) now supports recursion up to a depth of 1024.
+
+## CLI changes
+
+There are also several changes to the CLI:
+
+- `tofu init` now supports the `-json` flag for JSON output.
+- `tofu plan` now has a `-concise` flag to shorten the plan output.
+- `tofu console` now works on Solaris and AIX.
+- The CLI now supports the XDG directory specification.
+- Aliases for `state list` &rarr; `state ls`, `state mv` &rarr; `state move`, `state rm` &rarr; `state remove`.
```

Comment:
These may be nicer to show as a nested list instead of on one line. It feels a little cramped right now